RetroBar/Languages/Deutsch.xaml

```diff
@@ -49,16 +75,31 @@
     <s:String x:Key="quick_launch">Schnellstartleiste</s:String>
     <s:String x:Key="new_toolbar">Neue Symbolleiste...</s:String>
     <s:String x:Key="set_time">_Datum/Uhrzeit ändern</s:String>
+    <s:String x:Key="cascade">Fenste ka_skadieren</s:String>
+    <s:String x:Key="horiz_tile">Kachelfenster _Horizontal</s:String>
+    <s:String x:Key="vert_tile">Kachelfenster V_ertikal</s:String>
+    <s:String x:Key="toggle_desktop">De_sktop anzeigen</s:String>
+    <s:String x:Key="hide_desktop">Offene Fen_ster anzeigen</s:String>
+    <s:String x:Key="undo_cascade">Kaskadierung r_ückgängig machen</s:String>
+    <s:String x:Key="undo_tile">Kachel r_ückgängig machen</s:String>
+    <s:String x:Key="undo_minimize_all">Alles minimieren rü_ckgängig machen</s:String>
     <s:String x:Key="show_taskman_2k">_Task-Manager...</s:String>
     <s:String x:Key="show_taskman">_Task-Manager</s:String>
     <s:String x:Key="tray_properties">E_igenschaften</s:String>
     <s:String x:Key="update_available">Update verfügbar...</s:String>
     <s:String x:Key="exit_retrobar">Retrobar schließen</s:String>
     <s:String x:Key="customize_notifications_menu">Infobereich _anpassen...</s:String>
+    <s:String x:Key="lock_taskbar_menu">Die Taskleiste _fixieren</s:String>
+
+    <s:String x:Key="show_desktop_tip_95">Oberfläche/Desktop wiederherstellen</s:String>
+    <s:String x:Key="show_desktop_tip">Desktop anzeigen</s:String>
+    <s:String x:Key="show_desktop_95">_Desktop</s:String>
+    <s:String x:Key="show_desktop">De_sktop anzeigen</s:String>
+    <s:String x:Key="peek_at_desktop">_Blick auf Desktop werfen</s:String>
 
     <s:String x:Key="restore">_Wiederherstellen</s:String>
     <s:String x:Key="move">_Verschieben</s:String>
-    <s:String x:Key="size">_Größe ändern</s:String>
+    <s:String x:Key="size">_Größe</s:String>
     <s:String x:Key="minimize">Mi_nimieren</s:String>
     <s:String x:Key="maximize">Ma_ximieren</s:String>
     <s:String x:Key="close">_Schließen</s:String>
```

Comment:
Do some of these translations have to be changed or can they be kept?

![XPde1](https://github.com/dremin/RetroBar/assets/28744392/bd7197ad-53ac-4497-8d1b-50c38ed3de41)
![XPde2](https://github.com/dremin/RetroBar/assets/28744392/606602f0-7605-4cbe-bd73-00e3623b8099)

Reply:
I'm aware of that in original windows and changed some of the strings.

Where are the options from first screenshot in RetroBar?
They are not accessable with right click on the taskbar given by RetroBar.
![image](https://github.com/dremin/RetroBar/assets/51911097/426c115e-57d8-49b5-aef6-fc2b362743af)

cascade= Überlappend
horiz_tile = Untereinander
vert_tile = Nebeneinander
undo_cascade = ???
undo_tile = Nebeneinander rückgängig machen